app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -77,7 +77,7 @@ class ExplorationActivityTest {
 
   private fun createExplorationActivityIntent(explorationId: String): Intent {
     return ExplorationActivity.createExplorationActivityIntent(
-      ApplicationProvider.getApplicationContext(), explorationId
+      ApplicationProvider.getApplicationContext(), explorationId, null
```

Comment:
Provide inline comment for null over here.

Reply:
Done.